FEATURE_GATES.md

```diff
@@ -168,3 +168,89 @@ services: if two HTTPRoute rules use both serviceA and serviceB in their
 backendRefs, KIC will generate a single Kong service with endpoints from both
 serviceA and serviceB, named for the first rule and match indices with that
 combination of Services.
+
+## Using KongServiceFacade
+
+In KIC 3.1.0 we introduced a new feature called `KongServiceFacade`. Currently, we only
+support `KongServiceFacade` to be used as a backend for `netv1.Ingress`es.
```

Comment:
```suggestion
support `KongServiceFacade` to be used as a backend for `networking.k8s.io/v1` `Ingress`es.
```
`netv1` is jargon that I don't expect to be understandable by the general public :slightly_smiling_face: